source/d3d9/runtime_d3d9.cpp

```diff
@@ -271,6 +296,19 @@ namespace reshade::d3d9
 		}
 
 		_depth_source_table.clear();
+
+		// reset the new tables
+		_depth_buffer_table.clear();
+
+		if (!_depth_clearing_table.empty())
```

Comment:
This `if` is not necessary since the `for` loop doesn't do anything if the table is empty anyway.